### PR TITLE
Adds bad back quirk (makes bags give you a bad mood)

### PR DESCRIPTION
**Pull request by anconfuzedrock**
## About The Pull Request

Back pain is a -2 negative quirk that causes a _very_ bad moodlet when you wear a storage item on your back. Non storage items, like fire axes and picks, don't cause the mood.
## Why It's Good For The Game

First, I always felt 2d spacemen had too much storage space in those bags. It'd interest me to see people sacrifice some of that space and I think it'd be a far more engaging disability then deafness or paraplegics. Maybe briefcases would even see some use for people besides launchpad traitors! Maybe fanny packs would be seen at all!
Second, I feel that quite a few items that fit on the back never get any love because you're basically gimping yourself if you don't have a backpack on there. Have you ever seen someone wear a large airtank on their back? Combat shotguns are always carried in one hand, never on the back.
Third, It slightly bugs me that everyone has a strap across their chest or a completely obstructed back. Now maybe we'll be able to read the letters on the back of the blue letterman jacket or something.
## Changelog
:cl:
rscadd: New negative quirk - back pain causes very bad mood from wearing uneven storage items on your back, but other back items like air tanks and shotguns are okay.
/:cl:

Maybe I should change this thing's name to reverse hoarder or something, I dunno.

### CODE REVIEW

**Comment by OverlyGeneric** [comment]
I mean this would be good as its own mechanic for old characters but constant back pain is a whole lot worse.

It could mean degenerative spine disease, spinal disk disease, and simple things like infections.

**Comment by anconfuzedrock** [comment]
yeaah no I'm not gonna be the one to make age matter I'm going to be the one who gets 50% of his prs closed for being fucking dumb

**Comment by ATH1909** [comment]
Honestly, I think I'd prefer it if you just flat-out couldn't put storage items on your back (instead of just receiving a bad moodlet for them).

That said, I do like and approve of this PR.

**Comment by anconfuzedrock** [comment]
The bad moodlet is among the worst, to the point where wearing a bag with it is a retarded idea. The main reason I did it is to sort out how jobs start with a bag equipped.

File: code/datums/traits/negative.dm [hunk 1]
```diff
@@ -1,5 +1,19 @@
 //predominantly negative traits
 
+/datum/quirk/badback
+	name = "Bad Back"
+	desc = "Thanks to your poor posture, backpacks and other bags never sit right on your back. More evently weighted objects are fine, though."
+	value = -2
+	gain_text = "<span class='danger'>Your back REALLY hurts!</span>"
+	lose_text = "<span class='notice'>Your back feels better.</span>"
+
```
Comment (Mickyan) [inline code/datums/traits/negative.dm]: mood_quirk = TRUE

**Comment by JMoldy** [comment]
Would there be any way to make it check how full the backpack is so that you aren't stricken by horrible back pains for wearing an empty 1kg backpack but are perfectly fine carrying around an entire oxygen cylinder?

**Comment by anconfuzedrock** [comment]
No because that's not the point

**Comment by Bawhoppen** [comment]
Rename it to: "Ex-Mochilero"
"Due to years of exploitation through the traumatizing experience of trafficking space drugs, this character is highly resistant to ever wear any form of storage on their back again."

**Pull request merged**